third_party/BUILD

```diff
@@ -163,8 +163,10 @@ distrib_java_import(
     jars = [
         "api_client/google-api-client-1.22.0.jar",
         "api_client/google-api-client-jackson2-1.22.0.jar",
+        "api_client/google-api-client-gson-1.22.0.jar",
```

Comment:
Does that mean the newer version uses `gson` instead of `jackson2`? It would be nice if we can remove `jackson2`.

Reply:
yes looks like they switched to gson in 0.23.0 https://github.com/googleapis/google-auth-library-java/pull/531/files. Removed jackson dependency in pr